vreplgen/vreplgen.go

```diff
@@ -23,35 +23,79 @@ package main
 
 import (
 	"bytes"
+	"flag"
 	"fmt"
+	"os"
 	"strings"
 
 	"vitess.io/vitess/go/sqltypes"
 	binlogdatapb "vitess.io/vitess/go/vt/proto/binlogdata"
 )
 
+var onDdl string
+
+func init() {
+	flag.StringVar(&onDdl, "on_ddl", "ignore", "Set on_ddl value for replication stream - ignore, stop, exec, exec_ignore")
+	flag.Parse()
+}
+
 func main() {
-	vtctl := "./lvtctl.sh"
-	tabletID := "test-400"
-	dbName := "vt_merchant"
+	argOffset := 0
```

Comment:
You shouldn't have to use `argOffset`. It may be because you're calling `flag.Parse` in `init`. Can you move it into `main` and see if it works correctly?